cucumber/api/features/support/rest_helpers.rb

```diff
@@ -289,6 +289,10 @@ def random_hex nbytes = 12
 
   protected
 
+  def api_key_for(role)
```

Comment:
Can we call this `api_key_for_role` so that it's clear what the param is from the place where it's being called? Maybe even `api_key_for_role_name` since it's supposed to be a string and not a `Role` object.